player/command.c

```diff
@@ -2889,6 +2889,19 @@ static int mp_property_cursor_autohide(void *ctx, struct m_property *prop,
     return r;
 }
 
+static int mp_property_dvb_channel(void *ctx, struct m_property *prop,
+                                   int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    int r = mp_property_generic_option(mpctx, prop, action, arg);
+    if (action == M_PROPERTY_SET || action == M_PROPERTY_SWITCH) {
```

Comment:
There's a `is_property_set()` helper for this.

Reply:
Thanks, I totally missed that! 
Will change to use that, makes the intent much more readable.